### PR TITLE
Tidy features branch

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes part of #1906 

# 👩🏻‍💻 What does this PR do? 
 
Some minor tweaks to tidy the feature branch ahead of merging in.

## 📃 Documentation

_No user facing changes_

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "//": "The sed command here is a temporary workaround to fix an issue in graphql-codegen",
-    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec perl -i -pe 's/graphql-request\\/dist\\/types\\.dom/graphql-request\\/src\\/types\\.dom/g' {} \\;",
+    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml",
```
Comment (mark-prins) [inline client/package.json:23]: no longer required, and results in branch differences to the generated files. This is now handled by an alias in tsconfig.

File: client/packages/android/README.MD [hunk 2]
```diff
@@ -110,6 +110,8 @@ When adding plugins - add them to the android package; if they are not here, the
        @capacitor/preferences@4.0.2
 [info] Found 1 Cordova plugin for android:
        cordova-plugin-printer@0.8.0
+```
```
Comment (mark-prins) [inline client/packages/android/README.MD:113]: there's a start "```" but this end part was removed 🤷

File: client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ import {
   StockLineFragment,
   useStock,
 } from '@openmsupply-client/system';
-import { LocationSearchInput } from 'packages/system/src/Location/Components/LocationSearchInput';
+import { LocationSearchInput } from '../../..';
```
Comment (mark-prins) [inline client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx:14]: fixing imports for jest

**Review by andreievg — APPROVED** [review]
Thank you

**Pull request merged**